<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KoOpenPaneBase</class>
 <widget class="QWidget" name="KoOpenPaneBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>482</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTreeWidget" name="m_sectionList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_openExistingButton">
         <property name="text">
          <string>Open Existing Document</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QStackedWidget" name="m_widgetStack">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QWidget" name="WStackPage"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
